code/game/objects/items/ego_weapons/aleph.dm

```diff
@@ -1131,3 +1131,113 @@
 	if(B.safety)
 		user.remove_status_effect(STATUS_EFFECT_LCBURN)
 
+/obj/item/ego_weapon/mockery
+	name = "mockery"
+	desc = "...If I earned a name, will I get to receive love and hate from you? \
+	Will you remember me as that name, as someone whom you emotionally cared for?"
```

Comment:
```suggestion
	Will you remember me as that name, as someone whom you cared for?"
```
"emotionally cared for" reads weirdly here imo. "cared for" gets the same message across and I think reads better.

Reply:
It's a direct quote, but you have a point. You can't trust those translators.